### DISCUSSION TITLE
Registration via Google account. Mistake

### DISCUSSION THREAD

**Discussion by paolopa65** [greghesp/ha-bambulab#345] [Q&A]
Hi, I'm trying to log in to my Bambulab account on my A1.
I registered the account using Google login credentials.
When I enter my google credentials, email and password, I get rejected saying "Can't connect. Check your credentials".
The login credentials are the right ones, but I can't proceed.
I can't access via LAN because the Bambulab A1 doesn't have the password to connect via LAN.
How can I do?

**Comment by AdrianGarside** [comment]
Did you follow the instructions to add the password to bambu as described here? And do you have 2FA enabled? That's not supported.
https://github.com/greghesp/ha-bambulab/blob/main/README.md
